google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/AddBucketIamMember.java

```diff
@@ -32,24 +33,33 @@ public static void addBucketIamMember(String projectId, String bucketName) {
     // The ID of your GCS bucket
     // String bucketName = "your-unique-bucket-name";
 
+    // For more information please read:
+    // https://cloud.google.com/storage/docs/access-control/iam
     Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
-    Policy originalPolicy = storage.getIamPolicy(bucketName);
 
-    // See the StorageRoles documentation for other valid roles:
-    // https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/storage/StorageRoles.html
-    Role role = StorageRoles.objectViewer();
+    int policyVersion = 3;
```

Comment:
It got me thinking here - we want developers to use version `3` everywhere. Would it be better from that standpoint to get rid of this variable and directly use the value everywhere?

```java
...BucketSourceOption.requestedPolicyVersion(3);
```

```java
builder.setVersion(3);
```

Reply:
It was recommended by @kolea2 to use a separate variable. I'm okay with either but it's easier to understand what the value 3 means.

Reply:
I'll defer to you both here - please feel free to use whichever you think would make more sense to the user :)

Reply:
Following canonical samples on this one which aligns with @jkwlui.